src/pkg/scan/postprocessors/report_converters.go

```diff
@@ -60,7 +61,11 @@ func (c *nativeToRelationalSchemaConverter) ToRelationalSchema(ctx context.Conte
 		log.G(ctx).Infof("There is no vulnerability report to toSchema for report UUID : %s", reportUUID)
 		return reportUUID, "", nil
 	}
-
+	// to avoid the jobservice container oom, limit the size of the scan report data
+	if len(reportData) > reportDataSizeLimit {
+		log.Errorf("the scan report data too large to process, current size: %v bytes, limit size: %v bytes, will skip to process the scan report", len(reportData), reportDataSizeLimit)
```

Comment:
Here, error returns nil, so it might be more reasonable for the log to be a warning?